src/Components/Header/index.js

```diff
@@ -64,7 +64,8 @@ function MenuItem(index, selected, setSelected, title) {
 const styles = StyleSheet.create({
   logoContainer: {
     flex: 1,
-    marginRight:80,
```

Comment:
This is generic component and will change all the headers.

Reply:
@annabauza The issue which was meant to be solved in this PR is already solved in #176, so I think we should close this PR now?

Reply:
@annabauza  the issue was in the whole header so we have to change in the generic component for that and it's fixed.